storm-client/src/jvm/org/apache/storm/executor/spout/SpoutOutputCollectorImpl.java

```diff
@@ -172,7 +173,7 @@ public void reportError(Throwable error) {
             globalTupleInfo.setValues(values);
             globalTupleInfo.setMessageId(messageId);
             globalTupleInfo.setTimestamp(0);
-            globalTupleInfo.setId("0:");
+            globalTupleInfo.setRootId(0);
```

Comment:
This could be `globalTupleInfo.setRootId(rootId);` since rootId is 0 in this case. But I am fine with it

Reply:
Thanks, fixed.